Automattic/jetpack
# Blaze: Fixes a 404 error in the dashboard select posts page

## Proposed changes:


This fixes an issue in the Blaze dashboard - Select post to promote page, related to a 404 in the endpoint `blaze-app/sites/{SITE_ID}/wordads/dsp/api/v1/wpcom/sites/{SITE_ID}/blaze/posts`.

I am adding the missing controller inside the Blaze package, that will be in charge of redirecting that call to the WPCOM API.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:






On a self-hosted site, enable the Blaze Dashboard with `add_filter( 'jetpack_blaze_dashboard_enable', '__return_true' );`

* Go to Tools->Advertising
* Click on the Promote button next to the Jetpack Blaze header
* Click the Get Started button (if the get started page is shown for your user)
* Verify that the `Select post to promote` page is shown and that your posts eligible for Blaze are displayed.

![Screenshot 2023-06-29 at 13 36 24](https://github.com/Automattic/jetpack/assets/1258162/157eaf0c-2df9-46b9-a209-c4678a9c4c8d)
